common/include/pcl/common/io.h

```diff
@@ -368,10 +369,13 @@ namespace pcl
                   pcl::PCLPointCloud2 &cloud_out);
 
   /** \brief Check if two given point types are the same or not. */
-  template <typename Point1T, typename Point2T> inline bool
-  isSamePointType ()
+  template <typename T>
+  using expr_type = std::remove_cv_t<std::remove_reference_t<T>>;
```

Comment:
There's `remove_cvref_t` in pcl/type_traits.h

Reply:
Thx, used that